### PR TITLE
dom-querying - Junou Lee

### PR DESCRIPTION
**Pull request by junoulee**
image

### CODE REVIEW

File: dom-querying/main.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+console.log('hello, world');
+
+console.log(document.querySelector('h1'));
+console.dir(document.location);
```
Comment (codingcodymiller) [inline dom-querying/main.js:4]: What instruction told you to do this? 😕

File: dom-querying/main.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+console.log(document.querySelector('#explanation'));
+console.dir(document.location);
```
Comment (codingcodymiller) [inline dom-querying/main.js:6]: What instruction told you to do this? 😕

File: dom-querying/main.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+console.log(document.querySelector('.hint'));
+console.dir(document.location);
```
Comment (codingcodymiller) [inline dom-querying/main.js:8]: What instruction told you to do this? 😕

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: dom-querying/main.js (hunk 3 above)
Comment (junoulee) [reply dom-querying/main.js]: image

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**